Changes the maven group ID to match apache org

codefromthecrypt opened the pull request:
This also deletes publish configuration as that won't work.

See #2513

devinsba commented on pom.xml:
```diff
@@ -130,122 +133,135 @@
     <dependencies>
       <dependency>
-        <groupId>io.zipkin.zipkin2</groupId>
+        <groupId>org.apache.zipkin.zipkin2</groupId>
```
Maybe now is the time to drop the `zipkin2`, since my interpretation was that gave us the ability to have both on the classpath, this gives us that as well

codefromthecrypt replied:
drop for `org.apache.zipkin.zipkin` ? remember `org.apache.zipkin.brave` etc will exist. The main thing is not mingling other projects in the same group ID as that can be troublesome

devinsba commented on zipkin-autoconfigure/collector-kafka08/README.md:
```diff
@@ -13,14 +13,14 @@ JRE 8 is required to run Zipkin server.
 [executable jar for Zipkin server](https://search.maven.org/remote_content?g=io.zipkin.java&a=zipkin-server&v=LATEST&c=exec)
 and
-[autoconfigure module jar for the kafka collector](https://search.maven.org/remote_content?g=io.zipkin.java&a=zipkin-autoconfigure-collector-kafka08&v=LATEST&c=module).
+[autoconfigure module jar for the kafka collector](https://search.maven.org/remote_content?g=io.zipkin.zipkin2&a=zipkin-autoconfigure-collector-kafka08&v=LATEST&c=module).
```
```suggestion
[autoconfigure module jar for the kafka collector](https://search.maven.org/remote_content?g=org.apache.zipkin.zipkin2&a=zipkin-autoconfigure-collector-kafka08&v=LATEST&c=module).
```

codefromthecrypt replied:
there's a separate task for this, as this isn't published yet. #2513

devinsba commented on zipkin-autoconfigure/collector-kafka08/README.md:
```diff
@@ -13,14 +13,14 @@ JRE 8 is required to run Zipkin server.
 Fetch the latest released
 [executable jar for Zipkin server](https://search.maven.org/remote_content?g=io.zipkin.java&a=zipkin-server&v=LATEST&c=exec)
```
```suggestion
[executable jar for Zipkin server](https://search.maven.org/remote_content?g=org.apache.zipkin.zipkin2&a=zipkin-server&v=LATEST&c=exec)
```

codefromthecrypt replied:
there's a separate task for this, as this isn't published yet. #2513

devinsba commented on zipkin-autoconfigure/collector-scribe/README.md:
```diff
@@ -13,14 +13,14 @@ JRE 8 is required to run Zipkin server.
 Fetch the latest released
 [executable jar for Zipkin server](https://search.maven.org/remote_content?g=io.zipkin.java&a=zipkin-server&v=LATEST&c=exec)
```
```suggestion
[executable jar for Zipkin server](https://search.maven.org/remote_content?g=org.apache.zipkin.zipkin2&a=zipkin-server&v=LATEST&c=exec)
```

codefromthecrypt replied:
there's a separate task for this, as this isn't published yet. #2513

devinsba commented on zipkin-autoconfigure/collector-scribe/README.md:
```diff
@@ -13,14 +13,14 @@ JRE 8 is required to run Zipkin server.
 [executable jar for Zipkin server](https://search.maven.org/remote_content?g=io.zipkin.java&a=zipkin-server&v=LATEST&c=exec)
 and
-[autoconfigure module jar for the scribe collector](https://search.maven.org/remote_content?g=io.zipkin.java&a=zipkin-autoconfigure-collector-scribe&v=LATEST&c=module).
+[autoconfigure module jar for the scribe collector](https://search.maven.org/remote_content?g=io.zipkin.zipkin2&a=zipkin-autoconfigure-collector-scribe&v=LATEST&c=module).
```
```suggestion
[autoconfigure module jar for the scribe collector](https://search.maven.org/remote_content?g=org.apache.zipkin.zipkin2&a=zipkin-autoconfigure-collector-scribe&v=LATEST&c=module).
```

codefromthecrypt replied:
there's a separate task for this, as this isn't published yet. #2513

codefromthecrypt commented:
will rebase after this is merged, as that's why travis is busted https://github.com/apache/incubator-zipkin/pull/2521

Commit:
Changes the maven group ID to match apache org

This puts server assets under org.apache.zipkin and standalone libraries
under org.apache.zipkin.zipkin2

Commit:
Updated license files and removes date-specific config

codefromthecrypt commented:
@devinsba FYI I changed this to what you recommended.. org.apache.zipkin for everything, except org.apache.zipkin2 for libraries that could end up collocated
